packages/store/src/defineHeapKind.js

```diff
@@ -0,0 +1,72 @@
+// @ts-check
+
+import { Far } from '@endo/marshal';
+
+const { ownKeys } = Reflect;
+
+const becomePassable = (tag, thing) => {
```

Comment:
Even for internal use, this deserves at least a short comment. Its arguments can be of two different kinds, which are distinguished by sniffing, and handled differently.